### DISCUSSION TITLE
Proxy with serve.json doesn't work, vercel.json does.

### DISCUSSION THREAD

**Discussion by Jeremy-vdw** [vercel/vercel#8223] [Help] [answered]
Hi,

I want to run my application locally so I used the same json settings in the serve.json file as in the vercel.json file. On Vercel my application with the proxy works, but no locally. 

This are my settings:
```
{
    "rewrites": [
        {
            "source": "/api/:match*",
            "destination": "https://api.rentman.net/:match*"
        }
    ]
}
```

Can someone help me out please? I'm really stuck here.

**Comment by amyegan** [comment, marked as answer]
Hi @Jeremy-vdw. Are you using `vercel dev` to run the project locally, or a different command? Using [Vercel CLI](https://vercel.com/docs/cli) will let you replicate the Vercel deployment on your local machine. Otherwise, the `vercel.json` rewrites won't be picked up. More info about the dev command here https://vercel.com/docs/cli#commands/dev
